content/scripts/build.js

```diff
@@ -987,25 +1016,30 @@ class Builder {
 
     doc.title = metadata.title;
     doc.mdn_url = mdn_url;
-    if (metadata.parent) {
-      doc.parent = metadata.parent;
+    if (metadata.translation_of) {
+      doc.translation_of = metadata.translation_of;
     }
     doc.body = sections;
 
     doc.popularity = this.allPopularities[doc.mdn_url] || 0.0;
 
     doc.last_modified = metadata.modified;
 
-    const otherTranslations = this.allChildren[doc.mdn_url] || [];
-    if (!otherTranslations.length && metadata.parent) {
+    const otherTranslations = this.allTitles[doc.mdn_url].translations || [];
+    if (!otherTranslations.length && metadata.translation_of) {
       // But perhaps the parent has other translations?!
-      const parentMdnURL = buildMDNUrl(
-        metadata.parent.locale,
-        metadata.parent.slug
-      );
-      const parentOtherTranslations = this.allChildren[parentMdnURL];
-      if (parentOtherTranslations && parentOtherTranslations.length) {
-        otherTranslations.push(...parentOtherTranslations);
+      const parentURL = buildMDNUrl("en-US", metadata.translation_of);
+      const parent = this.allTitles[parentURL];
+      // See note in 'ensureAllTitles()' about why we need this if statement.
+      if (parent) {
+        const parentOtherTranslations = parent.translations;
```

Comment:
Nit. I think `parentData` better signifies the result than `parent`.
```suggestion
      const parentData = this.allTitles[parentURL];
      // See note in 'ensureAllTitles()' about why we need this if statement.
      if (parentData) {
        const parentOtherTranslations = parentData.translations;
```